docs/guides/functions.ipynb

```diff
@@ -73,7 +73,7 @@
    "id": "42843fcd-1779-4a74-8d6a-7d00c22e374e",
    "metadata": {},
    "source": [
-    "Optionally, you can use the `save_account()` method to save your credentials for easy access later on, before initializing the service. This saves your credentials in the same place as `QiskitRuntimeService.save_account()`, so you can skip this step if you had previously used `QiskitRuntimeService` to save your account. *Note that account credentials are saved in plain text, so only do so if you are using a trusted device.*"
+    "Optionally, you can use the `save_account()` method to save your credentials for easy access later on, before initializing the service. This saves your credentials in the same place as `QiskitRuntimeService.save_account()`, so you can skip this step if you had previously used `QiskitRuntimeService` to save your account. *Note that account credentials are saved in plain text. To more securely manage your token, it is recommended that you use an environment variable; see the [Securely manage your token](/guides/setup-channel#securely-manage) section for more information.*"
```

Comment:
I find this paragraph misleading. It seems to say using an env variable is more secure than using `save_account()`, which is not true because (1) both save in plaint text and (2) the instruction later uses `save_account()` anyway. 

I believe what we are trying to tell people to **avoid** using their API token in their notebooks ever. Hence I'd suggest:
1. Emphasize that one should never include their token in their notebooks
2. Recommend loading from an env variable _when doing save_account()_, so the token is not exposed even when doing this one-time setup. This should be part of the [setup process](https://docs.quantum.ibm.com/guides/setup-channel#set-up-to-use-ibm-quantum-platform) instead of a separate page. 
3. Change all the code examples that have `(token="<YOUR_IQP_API_TOKEN>")` to just `()`, perhaps with a note/link to the setup page if they haven't done so.